src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumPage.java

```diff
@@ -77,10 +70,10 @@ public ForumBoardPageObject openForumBoard() {
     wait.forElementClickable(forumBoardLink);
     scrollAndClick(forumBoardLink);
     PageObjectLogging.log("openForumBoard", "click on the forum Board", true, driver);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "openForumBoard" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)